tools/walletextension/wallet_extension.go

```diff
@@ -264,17 +259,17 @@ func (we *WalletExtension) handleEthJSON(userConn userconn.UserConn) {
 	}
 
 	respMap := make(map[string]interface{})
-	respMap[respJSONKeyID] = rpcReq.ID
-	respMap[respJSONKeyRPCVer] = jsonrpc.Version
-	respMap[RespJSONKeyResult] = rpcResp
+	respMap[JSONKeyID] = rpcReq.ID
+	respMap[accountmanager.JSONKeyRPCVersion] = jsonrpc.Version
```

Comment:
Minor thing but feels a bit weird to be getting some of these very standard RPC field names from a package called account manager.

Since they're standards maybe they can go in a common library or something, or be duplicated. Maybe they already exist in a standard lib?